src/kernels/jupyter/launcher/commandLineSelector.ts

```diff
@@ -36,7 +36,6 @@ export class JupyterCommandLineSelector {
         workspaceService.onDidChangeConfiguration(this.onDidChangeConfiguration.bind(this));
     }
 
-    @capturePerfTelemetry(Telemetry.SelectJupyterURI)
```

Comment:
This looks like it's been wrong forever. Glad we checked. Would have had too many entries for this event.